easybuild/easyconfigs/v/vbz_compression/vbz_compression-1.0.1-gompi-2020b.eb

```diff
@@ -0,0 +1,43 @@
+easyblock = 'CMakeMake'
+
+name = 'vbz_compression'
```

Comment:
This should be called `VBZ` (or, maybe, `VBZ_Compression`).

Reply:
done